mula/tests/integration/test_boefje_scheduler.py

```diff
@@ -258,6 +258,214 @@ def test_push_tasks_for_scan_profile_mutations_item_on_queue(
         self.assertEqual(task_db.id.hex, task_pq.id)
         self.assertEqual(task_db.status, models.TaskStatus.QUEUED)
 
+    @mock.patch("scheduler.schedulers.BoefjeScheduler.is_task_running")
+    @mock.patch("scheduler.schedulers.BoefjeScheduler.is_task_allowed_to_run")
+    @mock.patch("scheduler.schedulers.BoefjeScheduler.has_grace_period_passed")
```

Comment:
These lines are repeated for every test. If I'm not mistaken, you can also decorate the entire `TestCase` class.

Reply:
Picked up in https://github.com/minvws/nl-kat-coordination/issues/591